virtualcontext.js

```diff
@@ -32,9 +31,8 @@ function VirtualContext(id, endpoint=null, parent=null)
 	else
 	{
 		this.id = id || null;
-		this.endpoint = endpoint || null;
 		this.parent = parent || null;
-		this.properties = {"readonly": true};
+		this.properties = {"readonly": true, "virtualSrc": src};
```

Comment:
Why `virtualSrc` here if everywhere else was `src`?

Reply:
@mfmoreno understands that they have different meanings